packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -17,10 +17,9 @@ import '../template.dart';
 
 class CreateCommand extends FlutterCommand {
   CreateCommand() {
-    argParser.addFlag('pub',
-      defaultsTo: true,
-      help: 'Whether to run "pub get" after the project has been created.'
-    );
+
+    usesPubOption();
```

Comment:
This seems like a strange name for a function with side-effects, but it also doesn't look like you created this function.  @devoncarew